### PR TITLE
Updated the images and revamped the contents of the README.md

### PR DESCRIPTION
**Pull request by Giri-anandh**
The images are replaced with more details. Corrected the spelling and grammatical mistakes in the file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,54 +25,57 @@ Functions Palette -> Addons -> SLL Drona
...
+<ul>Stimulus is used to assign values to the UI controls based on the need and provide instructions, for the operations to be performed. This poly API available in the SLL Drona palette provides a set of stimulus VIs that help you in assigning the value for various types of control. </ul>
 
 ![Stimulus Control](Images/Stimulus%20API%20control.png) ![Stimulus Keyboard](Images/Stimulus%20API%20keyboard.png) ![Stimulus VI](Images/Stimulus%20API%20VI.png) <br/>
-<ul>You can set values for a control that are inside a tab control or a sub panel, etc by giving a proper valid control path and value for that control. The control path contains the label name of the controls with delimiters to access the specific or target control. </ul>
+<ul>Set values for a control that is inside a tab control or any other object by giving a proper valid control path and value for that control. The control path contains the label name of the controls with delimiters to access the specific control. </ul>
```
[the remark this answers is not in the document]
Comment (Giri-anandh) [reply README.md:49]: updated the changes

File: README.md [hunk 2]
```diff
@@ -25,54 +25,57 @@ Functions Palette -> Addons -> SLL Drona
...
-![Drona ini file](Images/Drona%20ini%20file.png) <br/>
-The above shown is the sample format required for creating Drona configuration file.
+If you are using Drona for a LabVIEW application, then the INI file has to be created parallel to it.</ul>
+ <b>Sample folder structure for the Drona INI file</b><br/>
+![Folder structure](Images/INI_File_Structure.png)
+<b>Sample format required for creating a Drona configuration file</b>
+![Drona ini file](Images/Drona%20ini%20file.png)
 ### 2. Stimulus:
-<ul>Stimulus is the step where you assign values to the UI controls based on the need and operations to perform. This poly API available in SLL Drona palette provides a set of stimulus VIs that helps you in assigning the value for various types of control. </ul>
+<ul>Stimulus is used to assign values to the UI controls based on the need and provide instructions, for the operations to be performed. This poly API available in the SLL Drona palette provides a set of stimulus VIs that help you in assigning the value for various types of control. </ul>
```
[the remark this answers is not in the document]
Comment (Giri-anandh) [reply README.md:46]: rephrased the sentences

File: README.md [hunk 3]
```diff
@@ -25,54 +25,57 @@ Functions Palette -> Addons -> SLL Drona
...
 
-You can find the related APIs to these steps in SLL Drona palette
+You can find the related APIs to these steps in the SLL Drona palette
 
 ### 1. Setup:
-<ul> First step in creating the test case is the setting up of the project and having a Drona configuration file inside the application instance. The Drona configuration file contains information such as the name of the Launcher VI that needs to be under UI test and whether to open and run it while launching the test. If you have configured the Drona file then you can just start creating the test case by dragging and dropping the Setup VI in block diagram which in return gives out the application reference and VI reference of the Launcher VI that needs to be tested. </ul>
+<ul> The first step in creating the test case is setting up the project and creating a Drona configuration file parallel to the Project. The configuration file includes the Launcher VI path, which is relative to the INI file, and options for opening and running the VI while executing the test. Test cases can be created after configuring the file. The Application reference and VI reference obtained from Setup VI are used to create test cases.
```
[the remark this answers is not in the document]
Comment (Giri-anandh) [reply README.md:38]: added explanation for the launcher VI

File: README.md [hunk 4]
```diff
@@ -25,54 +25,57 @@ Functions Palette -> Addons -> SLL Drona
...
-<ul>In this step you will assert the value obtained with the actual and restun whether the test is passed or failed. This poly API available in SLL Drona palette provides a set of assert VIs that helps you in asserting the values. </ul>
+<ul>This API can be used to assert the value of the controls and return whether the test is passed or failed. This poly API available in the SLL Drona palette provides a set of assert VIs that help in asserting the values. </ul>
 
 ![Assert Control Value](Images/Assert%20Control%20value.png) ![Assert Control Disable](Images/Assert%20Control%20disable.png) ![Assert Subpanel Inserted VI](Images/Assert%20Subpanel%20Inserted%20VI.png) ![Assert VI](Images/Assert%20VI%20Exec.png) ![Assert Control Visibility](Images/Assert%20Control%20visible.png) 
-<ul>As same as stimulus here also there needs to be a valid control path given. There are a set of Assert VIs available in SLL Drona palette which can be used to assert the value or obtained result. </ul> 
+<ul>As same as the stimulus a valid control path has to be given. There is a set of Assert VIs available in the SLL Drona palette which can be used to assert the value and obtain the results. </ul>
```
[the remark this answers is not in the document]
Comment (Giri-anandh) [reply README.md:61]: tried to make the wordings same for Stimulus and Assert

File: README.md [hunk 5]
```diff
@@ -25,54 +25,57 @@ Functions Palette -> Addons -> SLL Drona
...
-For example, To assert the value of the string in the below sample UI you need to give the input as <br/>
+For example, To assert the value for the string in the below sample UI the input has to be given as <br/>
 ![Assert control](Images/Assert%20control.png)<br/>
 *Tab Control>>Tab Control 2>>Cluster>>String* <br/>
 ![Assert workflow](Images/Assert%20workflow.png)
 ### 4. Cleanup:
-<ul> Close all the references that are opened while setting up the test case. For this just drag and drop the Close API in the Block diagram from SLL Drona palette and give the Launcher VI reference and application reference that are need to be closed. </ul>
+<ul> This VI closes all the references that are opened while setting up the test case. Provide this VI with the Launcher VI reference and application reference that need to be closed. </ul>
 
 ## Unit Tests
```
[the remark this answers is not in the document]
Comment (Giri-anandh) [reply README.md:70]: *Examples

File: README.md [hunk 6]
```diff
@@ -25,54 +25,57 @@ Functions Palette -> Addons -> SLL Drona
...
-You can find the examples for different use cases in the following path: <br/>
+Find the examples for different use cases in the following path: <br/>
 <*LabVIEW>\vi.lib\SLL Drona\Examples*  <br/>
-There are 4 different types of examples available here based on the use case.
-To get to know more about the use cases please visit the testcases under the following path: <br/>
+There are 5 different types of examples available here based on the use case.
```
[the remark this answers is not in the document]
Comment (Giri-anandh) [reply README.md:73]: rephrased the sentence

File: README.md [hunk 7]
```diff
@@ -82,13 +85,13 @@ To build the SLL Drona package, please visit this [document](BUILD.md) for more
...
 
-To contribute to SLL Drona, you need to have LabVIEW 2016 - 32 bit development environment.
+To contribute to SLL Drona, you need to have a LabVIEW 2016 32-bit development environment.
 
 ## License 
 SLL Drona is distributed under MIT [license](https://github.com/solitontech/SLL-Drona/blob/main/LICENSE). 
 
 ## Credits
-SLL Drona uses open source components. You can find the source code of their open source projects along with license information below. We acknowledge and are grateful to these developers for their contributions to open source. 
+SLL Drona uses open-source components. You can find the source code of their open-source projects along with license information below. We acknowledge and are grateful to these developers for their contributions to open source.
```
[the remark this answers is not in the document]
Comment (Giri-anandh) [reply README.md:95]: we have already provided the credits to OpenG right
image

File: README.md [hunk 8]
```diff
@@ -25,54 +25,58 @@ Functions Palette -> Addons -> SLL Drona
...
+You can find the related APIs to these steps in the SLL Drona palette
 
 ### 1. Setup:
-<ul> First step in creating the test case is the setting up of the project and having a Drona configuration file inside the application instance. The Drona configuration file contains information such as the name of the Launcher VI that needs to be under UI test and whether to open and run it while launching the test. If you have configured the Drona file then you can just start creating the test case by dragging and dropping the Setup VI in block diagram which in return gives out the application reference and VI reference of the Launcher VI that needs to be tested. </ul>
+<ul> The first step in creating the test case is setting up the project and creating a Drona configuration file parallel to the project. The configuration file includes the Launcher VI (used to start the program) path, which is relative to the INI file, and options for opening and running the VI while executing the test. Test cases can be created after configuring the file. The Application reference and VI reference obtained from Setup VI are used to create test cases.
```
Comment (Sairam0496) [inline README.md:38]: - [ ] ...Launcher VI (~~used to start the program~~ **the startup VI that launches the UI to be tested**) path...

File: README.md [hunk 9]
```diff
@@ -25,54 +25,58 @@ Functions Palette -> Addons -> SLL Drona
...
-*Tab Control>>Cluster>>Numeric* <br/>
+*Tab Control>>Cluster>>Numeric* <br/><br/>
+The label name has to be given as mentioned below. <br/>
 ![Stimulus workflow](Images/Stimulus%20workflow.png) <br/>
-As mentiond above you should give the label name of the controls and ensure whether its as same as the label name of the UI as its case sensitive. </ul>
+Make sure it is the same as the label name and is case-sensitive. </ul>
 
 ### 3. Assert:
-<ul>In this step you will assert the value obtained with the actual and restun whether the test is passed or failed. This poly API available in SLL Drona palette provides a set of assert VIs that helps you in asserting the values. </ul>
+<ul>Assert is used to verify the controls' value and return the test report. A collection of assert VIs are provided by this poly API, which can be found in the SLL Drona palette, to assist in asserting the values. </ul>
```
Comment (Sairam0496) [inline README.md:59]: - [ ] ...verify the controls' value**s, contents, properties, etc.** and...

File: README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (Sairam0496) [reply README.md:96]: - [ ] add latest repo link and license link

**Review by Sairam0496 — CHANGES_REQUESTED** [review]

File: README.md (hunk 9 above)
Comment (Giri-anandh) [reply README.md:59]: updated the line and removed the repetition

File: README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (Giri-anandh) [reply README.md:96]: updated the link to the openG

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Giri-anandh) [reply README.md]: updated the sentence again

**Review by Sairam0496 — APPROVED** [review]
All good! Approving hoping images will be raised as separate PR.

**Comment by Giri-anandh** [comment]
> All good! Approving hoping images will be raised as separate PR.

All the images have been included in this PR itself sai!!

**Comment by Sairam0496** [comment]
Was mentioning this change based on Assert API Change
image

Will this be coming under the PR for Assert Poly change?

**Pull request merged**